src/components/nav.component.astro

```diff
@@ -3,6 +3,7 @@ import InfoIcon from './icons/info.icon.astro';
 import DdjsLogo from './icons/ddjs-logo.icon.astro';
 import MeetupLogo from './icons/meetup-logo.icon.astro';
 import SlackLogo from './icons/slack-logo.icon.astro';
+import ThemeSwitch from './theme-switch.component.astro';
 
 const { pathname } = Astro.url;
 const imprintLink = `${import.meta.env.BASE_URL}imprint`;
```

Comment:
Instead of using Nodes meta, we should stick to the props Astro is providing through it's API.

Reply:
Done